vizro-core/src/vizro/models/_navigation/_accordion.py

```diff
@@ -65,6 +65,10 @@ def _create_default_accordion(self):
 
         accordion_items = [self._create_accordion_item(accordion_buttons=accordion_buttons)]
 
+        # Don't create accordion navigation if there is only one page and one accordion item
+        if len(accordion_buttons) == len(accordion_items) == 1:
+            return None
```

Comment:
Should we enable user to create one page navigation if they explicitly define navigation in the dashboard?

Reply:
No, I don't think we should. It's a visual guideline enforced by us. The navigation doesn't have any functional purpose if there is only one page. It only adds visual noise which we should prevent from a design perspective. 

However, if for some reason many people request it, we can always add it in, but I would not enable it now